task4.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+function GetResultOfTask4(){
+    let array = [
+        {value: 100, type: 'USD'},
+        {value: 215, type: 'EUR'},
+        {value: 7, type: 'EUR'},
+        {value: 99, type: 'USD'},
+        {value: 354, type: 'USD'},
+        {value: 12, type: 'EUR'},
+        {value: 77, type: 'USD'},
+    ];
+    let sum = getSumOfUSD(array);
+    let showArray = getArrayOfEUR(array);
+    let message = `Пункт 1:`+sum + "<br/>Пункт 2:<br/>" + showArray;
+    document.getElementById('OutputArray').innerHTML = message;
+}
+
+function getSumOfUSD(array) {
+    let sum = 0;
+    array.forEach(element=>{
```

Comment:
Hello there)) Could you please use here Array.prototype.filter and Array.prototype.reduce instead of Array.prototype.forEach?